imports/lib/models/User.ts

```diff
@@ -45,6 +45,7 @@ export const User = z.object({
   displayName: nonEmptyString.optional(),
   googleAccount: nonEmptyString.optional(),
   googleAccountId: nonEmptyString.optional(),
+  googleProfilePicture: nonEmptyString.optional(),
```

Comment:
should we document anywhere that this won't be populated for existing linked accounts? I assume we don't particularly care about backfilling (e.g. on the next sign-in for that user?)